script/hatchet.ini

```diff
@@ -1,73 +1,77 @@
 [run]
 # What individual steps of HATCHet should we run in the pipeline?
 # Valid values are True or False
-download_panel=True
-count_reads=True
-genotype_snps=True
-phase_snps=True
-fixed_width=False # True uses older fixed-width versions of some commands
-count_alleles=True
-combine_counts=True
-cluster_bins=True
-loc_clust=True # True uses new locality-aware clustering
-plot_bins=True
-compute_cn=True
-plot_cn=True
+download_panel = True
+count_reads = True
+genotype_snps = True
+phase_snps = True
+fixed_width = False  # True uses older fixed-width versions of some commands
+count_alleles = True
+combine_counts = True
+cluster_bins = True
+loc_clust = True  # True uses new locality-aware clustering
+plot_bins = True
+compute_cn = True
+plot_cn = True
+
+# What chromosome(s) do we wish to process in the pipeline? Leave unspecified to process
+# all chromosomes found in the normal/tumor bam files
+chromosomes =
 
 # Path to reference genome
 # Make sure you have also generated the reference dictionary as /path/to/reference.dict
-reference = "/path/to/reference.fa"
```

Comment:
Quotations are not needed for string types (and are not used in any place unless necessary to interpret numbers as strings)